SU-HSDO/suhumsci
# HSD8-392 Added setting to float image to the right

# READY FOR REVIEW

# Summary
- Allow user to float layout image to the right.

# Needed By (Date)
- 10/10

# Urgency
- low

# Steps to Test

1. Checkout this branch
1. clear cache
1. create a basic page
1. edit the layout
1. choose "3 column with image" layout and set image to float left
1. populate the regions
1. validate it looks as you intended
1. change the image to float to the right.
1. validate it looks as you'd expect.

# See Also
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)
